applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+                    <doc>
+                        This field can be two things:
+
+                        1. For a pixel detector it provides the nominal wavelength
+                        for which the detector has been calibrated.
+
+                        2. For other detectors this field has to be seen together with
+                        the efficiency field above.
+                        For some detectors, the efficiency is wavelength dependent.
+                        Thus this field provides the wavelength axis for the efficiency field.
+                        In this use case, the efficiency and wavelength arrays must
+                        have the same dimensionality.
+                    </doc>
+                    <dimensions rank="2">
+                      <dim index="1" value="i" />
+                      <dim index="2" value="j" />
+                    </dimensions>
+                </field>
+                <field name="dead_time" type="NX_FLOAT" units="NX_TIME" minOccurs="0" maxOccurs="1">
+                    <doc>Detector dead time</doc>
+                    <dimensions rank="3">
+                        <dim index="1" value="nP" />
+                        <dim index="2" value="i" />
+                        <dim index="3" value="j" />
+                    </dimensions>
+                </field>
+                <field name="count_time" type="NX_NUMBER" units="NX_TIME" minOccurs="0" maxOccurs="1">
+                    <doc>Elapsed actual counting time</doc>
+                    <dimensions rank="1">
+                        <dim index="1" value="nP" />
+                    </dimensions>
+                </field>
+                <field name="depends_on" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>
+                        The axis on which the detector position depends may be stored
+                        anywhere, but is normally stored in the *NXtransformations
+                        group* within the *NXdetector group*.
+                    </doc>
+                </field>
+                <group type="NXtransformations" minOccurs="0" maxOccurs="1">
+                    <doc>
+                        This is the recommended location for detector goniometer
+                        and other related axes.
+                    </doc>
+                </group>
+            </group>
+            <group name="beam_intensity_profile" type="NXbeam" minOccurs="1" maxOccurs="1">
+                <doc>
+                 Defines the dimensions of the beam profile used for probing the sample which corresponds to or can be used to determine the instrumental gauge volume.  
+                 A description of the subsequent fields can be found in the folowing figure. The term "primary" in the subsequent fields refers to the beam path between the sample and the source. The term "secondary" refers to the beam path between the sample and the detector(s). 

```

Comment:
This changes NXbeam to sample two locations instead of

Reply:
Sorry, can you clarify? You mean this NXbeam should be under NXsample?

Reply:
I think you mean that primary and secondary should be two NXbeam instances?

Reply:
Too late to change...